Unified/completor.py

```diff
@@ -19,6 +19,9 @@ def forceComplete(url, wfi):
             #sendEmail("force completing","%s is worth force completing\n%s"%( member['RequestName'] , percent_completions))
             print "setting",member['RequestName'],"force-complete"
             reqMgrClient.setWorkflowForceComplete(url, member['RequestName'])
+        elif member['RequestStatus'] in ['acquired','assignment-approved']:
```

Comment:
reqMgrClient.invalidateWorkflow takes care of switching from aborted to rejected. something we could have in reqmgr2 to do it transparently.